Content/Items/Gear/Weapons/Wand/Wand.cs

```diff
@@ -8,9 +8,10 @@ namespace PathOfTerraria.Content.Items.Gear.Weapons.Wand;
 internal class Wand : Gear
 {
 	public override string Texture => $"{PathOfTerraria.ModName}/Assets/Items/Gear/Weapons/Wand/WandPlaceholder";
-
 	public override float DropChance => 1f;
 
+	protected override string GearLocalizationCategory => "Wand";
```

Comment:
Should we have this set in `Gear` and use GetType().Name?

Reply:
That is already the case. The thing is, given Sword and Broadsword, Sword is the only one with a category - as both share their prefixes/suffixes - since it wouldn't make sense to define localization for Sword and Broadsword which are the same.